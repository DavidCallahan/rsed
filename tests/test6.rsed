skip to "/asx/"
replace all "/TARGETS" "/BUCK"

