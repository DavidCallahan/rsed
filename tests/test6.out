admarket/asx/common/test/BUCK:cpp_benchmark(
admarket/asx/common/test/BUCK-    name='ngram_bench',
admarket/asx/common/test/BUCK-    srcs=[
--
admarket/cpe/common/tests/BUCK:cpp_benchmark(
admarket/cpe/common/tests/BUCK-    name='cpe_laser_client_benchmark_test',
admarket/cpe/common/tests/BUCK-    srcs=[
admarket/lib/cache/BUCK:cpp_benchmark(
admarket/lib/cache/BUCK-  name = "generic_lru_cache_benchmark",
admarket/lib/cache/BUCK-  srcs = [ "GenericLruCacheBenchmark.cpp" ],
--
admarket/lib/data/BUCK:cpp_benchmark (
